NewSpring/ApollosProject
# [DEV-4210] Add Staff News to the For Staff section of the Home Feed

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR removes the `Rock - All Staff News` content channel from the "For Staff" section of the home feed and replaces it with the `NewSpring - Staff News` content channel items. It also sets us up in case we ever want to use a content channel that is prefixed with "Rock - ". Lastly, it sets up Android to allow sharing the staff news items using the `/staff/news` route.

### How do I test this PR?
Fire up that sim.
Make sure the items in the `NewSpring - Staff News` content channel show up in the "For Staff" section on the home feed and at the top of the Discover feed.
Make sure that the URL that gets shared takes the form of `https://newspring.cc/staff/news/example-content-item-slug`. This should work right now on Android. You can test this by: hitting the share button on one of the staff news items, closing the app, pasting the link in messages, then tapping that link. The app should reopen and the content item you shared should open up. There will be a separate Rock change for the `apple-app-site-association` file.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
